frontend/src/components/Modal/styles.scss

```diff
@@ -46,6 +63,35 @@ input {
     font-size: 17px;
 }
 
+.close-button {
+    outline: none !important;
+    background-color: red;
```

Comment:
Nit: Maybe swapping this into `:hover` so this is red upon hover but grey otherwise?